From 836111a314b1d3f2763f1b3858511994f3ad88b9 Mon Sep 17 00:00:00 2001
From: Deepak Rathore <deepak.rathore@einfochips.com>
Date: Thu, 20 Jan 2022 17:18:13 +0530
Subject: [PATCH] machine: packages required for ap1302 interface are added

following packages are appended for ap1302 ISP interface
1. av96-root-files
2. ap1302-firmware

Signed-off-by: Deepak Rathore <deepak.rathore@einfochips.com>
---
 conf/machine/dh-stm32mp1-dhcor-avenger96.conf | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/conf/machine/dh-stm32mp1-dhcor-avenger96.conf b/conf/machine/dh-stm32mp1-dhcor-avenger96.conf
index 1c0495f..a4c13c5 100644
--- a/conf/machine/dh-stm32mp1-dhcor-avenger96.conf
+++ b/conf/machine/dh-stm32mp1-dhcor-avenger96.conf
@@ -27,3 +27,7 @@ MACHINE_ESSENTIAL_EXTRA_RDEPENDS += " \
 	cypress-firmware-bcm43xx-fmac-fw-bcm43455-1mw-sdio \
 	cypress-firmware-bcm43xx-fmac-nvram-bcm43455-1mw-sdio \
 	"
+
+#Changes for ap1302 interface
+IMAGE_INSTALL_append += "ap1302-firmware \
+                         av96-root-files "
-- 
2.17.1

